examples/llama.android/llama/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ FetchContent_MakeAvailable(llama)
 # used in the AndroidManifest.xml file.
 add_library(${CMAKE_PROJECT_NAME} SHARED
     # List C/C++ source files with relative paths to this CMakeLists.txt.
-    llama-android.cpp)
+        llama-android.cpp)
```

Comment:
was this tab here intentional?

Reply:
no, androidstudio must have done it

Reply:
Ah fair enough. If it looks better with the space in your opinion then that's fine.

CI is throwing up some error so you got some opportunities to fix that nitpick if you feel like it.

Reply:
The CI is likely failing because the Android project pull `llama.cpp` from the `master` branch:

https://github.com/ggerganov/llama.cpp/blob/e0f556186b6e1f2b7032a1479edf5e89e2b1bd86/examples/llama.android/app/src/main/cpp/CMakeLists.txt#L9-L21

So it is mixing changes from this PR and what is currently on `master`. Not familiar enough with Android build system to figure out how to fix this

Let's merge and see if it works